bin/stack-helpers.sh

```diff
@@ -1,11 +1,12 @@
-set -o pipefail
+# shellcheck shell=bash
```

Comment:
This is a helper script that is sourced, so doesn't need a shebang/bash mode setting.